packages/common/types/config.ts

```diff
@@ -39,20 +41,20 @@ export type Drm = {
   defaultPolicyId: string;
 };
 
-export type ContentType = 'playlist' | 'continue_watching' | 'favorites';
+export type PlaylistType = keyof typeof PLAYLIST_TYPE;
 
 export type Content = {
   contentId?: string;
   title?: string;
-  type: ContentType;
+  type: PlaylistType;
   featured?: boolean;
   backgroundColor?: string | null;
 };
 
 export type Menu = {
   label: string;
   contentId: string;
-  type?: Extract<ContentType, 'playlist'>;
+  type?: Extract<PlaylistType, 'playlist' | 'content_list'>;
```

Comment:
This can be easier since you have a type for it now:
```suggestion
  type?: PLAYLIST_TYPE.playlist | PLAYLIST_TYPE.content_list;
```